pages/landing/index.js

```diff
@@ -40,34 +25,6 @@ function Landing({ changeTheme }) {
     };
   }, []);
 
-  const onPoolSelectChange = (event, theOption) => {
-    setAsset(theOption);
-
-    setLoading(true);
-    stores.dispatcher.dispatch({
-      type: GET_MAX,
-      content: { address: theOption.address },
-    });
-  };
-
-  const renderAssetOption = (web3, option) => {
```

Comment:
I have my doubts, why this got introduced in 2ba42bd85a2151eeb763f7188b2f567022a82870 actually.
Did we plan to use it later on?
If we did then maybe we should explain this in a comment and add a linter exception.
Otherwise it's a bit confusing to see some dead code

Reply:
I think this was a case of Copy Paste and not removing the unused code.